components/Accordion/AccordionItem.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```

Comment:
I think we use should also use the same Compound Components pattern here. Because the accordion Component is not flexible with this structure. It works for data now. 


`
<Accordion>
        <Accordion.Item> Test Accordion </Accordion.Item> 
        <Accordion.Detail>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
        </Accordion.Details>
</Accordion>
`